docs/quickstart/docker.md

```diff
@@ -58,21 +58,28 @@ vi `/etc/docker/daemon.json`, Add `tcp://0.0.0.0:2375` to the `hosts` configurat
 
 ## Quickstart
 
-Modify these 2 configuration items in `zeppelin-site.xml`.
+1. Modify these 2 configuration items in `zeppelin-site.xml`.
 
 ```
-  <property>
-    <name>zeppelin.run.mode</name>
-    <value>docker</value>
-    <description>'auto|local|k8s|docker'</description>
-  </property>
-
-  <property>
-    <name>zeppelin.docker.container.image</name>
-    <value>apache/zeppelin</value>
-    <description>Docker image for interpreters</description>
-  </property>
+<property>
+  <name>zeppelin.run.mode</name>
+  <value>docker</value>
+  <description>'auto|local|k8s|docker'</description>
+</property>
+
+<property>
+  <name>zeppelin.docker.container.image</name>
+  <value>apache/zeppelin</value>
+  <description>Docker image for interpreters</description>
+</property>
+```
+
+2. set timezone in zeppelin-env.sh
 
+Set to the same time zone as the zeppelin server, keeping the time zone in the interpreter docker container the same as the server. E.g, `"America/New_York"` or `"Asia/Shanghai"`
+
+```
+export DOCKER_TIME_ZONE="America/New_York"
```

Comment:
Is there any default value?

Reply:
This is the world's various time zones, a generic string, not our custom string.